File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SylvanMandate.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Dagger;

/* JADX INFO: loaded from: classes3.dex */
public class SylvanMandate extends Dagger {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Dagger, it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Weapon
    public boolean isMagic() {
        return true;
    }

    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.weapon_dagger_sylvan_mandate_name;
        this.idDescription = R.string.weapon_dagger_sylvan_mandate_description;
        this.idEffect = R.string.weapon_dagger_sylvan_mandate_effect;
        this.idImage = R.drawable.sylvan_mandate;
        this.price = 12006L;
        this.intelligence = 50;
        this.dexterity = 30;
        this.constitution = 30;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SylvanMandate.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Dagger

class SylvanMandate : Dagger() {
    override fun configureProperties() {
        idName = R.string.weapon_dagger_sylvan_mandate_name
        idDescription = R.string.weapon_dagger_sylvan_mandate_description
        idEffect = R.string.weapon_dagger_sylvan_mandate_effect
        idImage = R.drawable.sylvan_mandate
        price = 12006L
        intelligence = 50
        dexterity = 30
        constitution = 30
    }

    override fun isMagic(): Boolean = true
}
